configuration/configuration.go

```diff
@@ -142,3 +152,66 @@ func GetHTTPAddress() string {
 func IsPostgresDeveloperModeEnabled() bool {
 	return viper.GetBool(varDeveloperModeEnabled)
 }
+
+func getRSAPrivateKey() string {
```

Comment:
Why are these new functions not exported? The name should start with a capital letter.